src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -323,7 +323,12 @@ public void run()
             }
           }
           else {
-            register(channel, SelectionKey.OP_CONNECT, l);
+            /*
```

Comment:
I saw it, note that the bug was closed as "not an issue". I also tried to register OP_CONNECT before connection request is made and it did not help.